specifications/drtm-handler-and-table.rst

```diff
@@ -0,0 +1,276 @@
+DLE Handler Specification
+=========================
+
+This specification details the state the platform should be in when invoking a DLE Handler.
+
+Overview
+--------
+
+The DLE (Dynamic Launch Event) Handler consumes the TrenchBoot DRTM Table and invokes the platform's dynamic launch event. This specification seeks to standardize the invocation interface for the DLE Handler for each platform supported by TrenchBoot. This will provide a well-defined interface for DLE Handler implementors to follow and bootloader to comply to ensure interoperability between implementations.
+
+Acronyms
+--------
+
+DCE - Dynamic Configuration Environment (Intel ACM/AMD SLB)
+DLE - Dynamic Launch Event (Intel GETSEC[SENTER]/AMD SKINIT)
+DLE Handler - Dynamic Launch Event Handler
+DLME - Dynamic Launch Measured Environment (Operating System/Hypervisor)
+DRTM - Dynamic Root of Trust Measurement
+SL Entry - Secure Launch Entry (Secure Launch Stub for Linux)
+
+Platform Requirements
+---------------------
+
+1 - x86 Platforms
+1.1 - The DLE Handler will be invoked with the CPU in 64bit long mode 
+1.2 - The DRTM Table will be passed to the DLE Handler in the RAX CPU register
+
+2 - Arm Platforms
+2.1 - TBD
+
+
+DRTM Table Specification
```

Comment:
Not sure where would be best, but I would add that it should not be confused with DRTM Resources Table as defined in https://trustedcomputinggroup.org/wp-content/uploads/TCG_D-RTM_Architecture_v1-0_Published_06172013.pdf. Maybe we should use different name to avoid the confusion?

Reply:
Do you think calling it the "Secure Launch DRTM Table" would be enough, or are you thinking that DRTM should not be in the name at all?

Are there any alternative names that anyone think might be better?

Reply:
+1 on the suggested name from me, but maybe someone else has other ideas.